src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -18,6 +18,8 @@ kibana_vars=(
     console.enabled
     console.proxyConfig
     console.proxyFilter
+    ops.cGroupOverrides.cpuPath
+    ops.cGroupOverrides.cpuAcctPath
     cpu.cgroup.path.override
     cpuacct.cgroup.path.override
```

Comment:
I added the new ones, but also kept the deprecated name to avoid a breaking change